### ISSUE TITLE
Migration issue SQLite

### ISSUE THREAD

**Issue by john-yick** [mikro-orm/mikro-orm#460]
There seems to be an issue with doing a migration against SQLite from using the demo code to migrate. https://mikro-orm.io/docs/migrations 

The migration file does seem to create correctly, but it does not seem to apply to the SQLite database.

**Stack trace**
```
[Nest] 16208   - 04/06/2020, 15:58:29   [MikroORM] [info] MikroORM successfully connected to database ./data/db.sqlite3
[Nest] 16208   - 04/06/2020, 15:58:29   [MikroORM] [query] select name as table_name from sqlite_master where type = 'table' and name != 'sqlite_sequence' and name != 'geometry_columns' and name != 'spatial_ref_sys' union all select name as table_name from sqlite_temp_master where type = 'table' order by name [took 0 ms]
[Nest] 16208   - 04/06/2020, 15:58:29   [MikroORM] [query] select name as table_name from sqlite_master where type = 'table' and name != 'sqlite_sequence' and name != 'geometry_columns' and name != 'spatial_ref_sys' union all select name as table_name from sqlite_temp_master where type = 'table' order by name [took 0 ms]
[Nest] 16208   - 04/06/2020, 15:58:29   [MikroORM] [query] begin
[Nest] 16208   - 04/06/2020, 15:59:29   [MikroORM] [query] select * from `mikro_orm_migrations` order by `id` asc [took 60007 ms]
[Nest] 16208   - 04/06/2020, 15:59:29   [MikroORM] [query] rollback
(node:16208) UnhandledPromiseRejectionWarning: KnexTimeoutError: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
    at Client_SQLite3.acquireConnection (/Users/johnyick/Documents/code/gitlab/nestjs-boilerplate/node_modules/knex/lib/client.js:347:26)
```

**To Reproduce**
Create some entities which all seem to discover correctly and run a migration though npm.
```
  "migrate": "ts-node migrate.ts"
```
`npm run migrate`

migrate.ts
`
import { MikroORM } from 'mikro-orm';
import Config from './src/mikro-orm.config';

(async () => {
    const orm = await MikroORM.init(Config);

    const migrator = orm.getMigrator();
    await migrator.createMigration(); // creates file Migration20191019195930.ts
    await migrator.up(); // runs migrations up to the latest

    await orm.close(true);
})();
`

**Expected behavior**
Migration should create a new migration file and apply the migration to the database.

**Additional context**

**Versions**
 "mikro-orm": "^3.6.1",
"typescript": "^3.7.4",

**Comment by B4nan** [comment]
This should be fixed in 3.6.2 (#444).

**Comment by john-yick** [comment]
@B4nan confirmed fixed in 3.6.2 thanks for the quick follow-up
